### DISCUSSION TITLE
关于 MagicJS

### DISCUSSION THREAD

**Discussion by xzxiaoshan** [blackmatrix7/ios_rule_script#637] [General]
作者你好，MagicJS 有没有独立的维护地址，平时基于MagicJS写脚本的话，如何获得和更新对应的最新版？谢谢~

**Comment by blackmatrix7** [comment]
目前MagicJS有段时间没有维护了，一些客户端的新特性也没有支持，目前的功能仅能满足我的使用需求。

所以暂时没有单独的项目，如果有意一起维护的话， 可以单独开一个项目。

**Reply by xzxiaoshan** [reply]
我是后端开发，Java的，前端不太熟，做起来应该问题也不大。
MagicJS 我做了格式化看了下，感觉是一个很不错的封装整理。
目前玩脚本的基本上都需要对Loon、Qx 这些进行操作，从技术本质上来讲，感觉即刻把它独立出来为一个库很合适。
我相信也有其他小伙伴会共同添砖加瓦贡献代码的。
所以我认为独立出来挺好。

**Reply by blackmatrix7** [reply]
https://github.com/blackmatrix7/MagicJS

我把它独立出来了，有兴趣可以一起维护。

目前代码还有很多需要改进的地方。

**Comment by xzxiaoshan** [comment]
👌
